Sources/ResultExtensions.swift

```diff
@@ -11,3 +11,15 @@ import Result
 internal func materialize<T>(_ f: () throws -> T) -> Result<T, AnyError> {
 	return materialize(try f())
 }
+
+extension Result: SignalProducerConvertible {
+	public var producer: SignalProducer<Value, Error> {
+		switch self {
```

Comment:
You can use the initializer here: https://github.com/ReactiveCocoa/ReactiveSwift/blob/55ac251953f0d0fe0076d7b7a5b9a3c2b9c382bb/Sources/SignalProducer.swift#L156

Reply:
Done 👍